app/controllers/concerns/sufia/batch_uploads_controller_behavior.rb

```diff
@@ -6,25 +6,39 @@ module BatchUploadsControllerBehavior
 
     included do
       self.work_form_service = BatchUploadFormService
-      self.curation_concern_type = work_form_service.form_class.model_class
+      self.curation_concern_type = work_form_service.form_class.model_class # includes CanCan side-effects
+      # We use BatchUpload as a null stand-in curation_concern_type.
+      # The actual permission is checked dynamically during #create.
     end
 
+    # The permissions to create a batch are not as important as the permissions for the concern being batched.
+    # @note we don't call `authorize!` directly, since `authorized_models` already checks `user.can? :create, ...`
     def create
       authenticate_user!
-      create_update_job
+      unsafe_pc = params.fetch(:batch_upload, {})[:payload_concern]
```

Comment:
This is part of the underlying `batch_create_work_type` branch and will fall out after that gets merged.  You might have a point, but its applicability would be on #2970.